src/DssSpell.sol

```diff
@@ -38,13 +48,85 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
     // uint256 internal constant X_PCT_RATE      = ;
+    uint256 internal constant PT_TWO_FIVE_PCT_RATE = 1000000000079175551708715274;
```

Comment:
Last usage of this value is on 2022-02-02 in Archives and refers to it as `ZERO_PT_TWO_FIVE_PCT_RATE` rather than `PT_TWO_FIVE_PCT_RATE` (semantics here tbh) - what do you prefer?

Reply:
Looking at the archive I can't find an example of ` PT_...` so lets change this to `ZERO_PT_...`

```suggestion
    uint256 internal constant ZERO_PT_TWO_FIVE_PCT_RATE = 1000000000079175551708715274;
```